traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -497,6 +498,12 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{api.Version{3, 0}, http.MethodGet, `letsencrypt/dnsrecords/?$`, deliveryservice.GetDnsChallengeRecords, auth.PrivLevelOperations, Authenticated, nil, 2534390553, noPerlBypass},
 		{api.Version{3, 0}, http.MethodPost, `letsencrypt/autorenew/?$`, deliveryservice.RenewCertificates, auth.PrivLevelOperations, Authenticated, nil, 2534390563, noPerlBypass},
 
+		// Acme account information
+		{api.Version{3, 0}, http.MethodGet, `acme/accounts/?$`, acme.Read, auth.PrivLevelAdmin, Authenticated, nil, 2034390561, noPerlBypass},
+		{api.Version{3, 0}, http.MethodPost, `acme/accounts/?$`, acme.Create, auth.PrivLevelAdmin, Authenticated, nil, 2034390562, noPerlBypass},
+		{api.Version{3, 0}, http.MethodPut, `acme/accounts/?$`, acme.Update, auth.PrivLevelAdmin, Authenticated, nil, 2034390563, noPerlBypass},
+		{api.Version{3, 0}, http.MethodDelete, `acme/accounts/{provider}/{email}?$`, acme.Delete, auth.PrivLevelAdmin, Authenticated, nil, 2034390564, noPerlBypass},
```

Comment:
If `/acme` doesn't exist, then I think this should just be `/acme_accounts`

Reply:
Also, these have to go in at least API 3.1 unless we're going to hold up 5.0.0 for this PR (which I don't really wanna do)